packages/sources/src/styling/variables/zindex.scss

```diff
@@ -1,5 +1,31 @@
-$zindex-navbar: 1000;
-$zindex-dropdown: 1000;
-$zindex-popover: 1060;
-$zindex-tooltip: 1070;
-$zindex-navbar-fixed: 1030;
+// Selectors
+$zx-viewer-grid-sizing-buttons:     -1;
+$zx-viewer-grid-map-panel:          100;
+
+$zx-selectable:                     100;
+
+$zx-datepicker-base:                100;
+$zx-datepicker-dialog:              210;
+
+$zx-description-open:               100;
```

Comment:
ik twijfel tussen `$zx-description-open` en gewoon `$zx-description`, maar kan me wel in jouw keuze vinden hoor. wat was jouw motivatie?

Reply:
Het gaat specifiek om de before en after die gerelateerd zijn aan de description die open is. Ik vind `$zx-description` dan te generiek